src/Lecture2.hs

```diff
@@ -97,7 +115,11 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+
+
+dropSpaces :: [Char] -> [Char]
+dropSpaces = dropWhileEnd isSpace . dropWhile isSpace
```

Comment:
I believe this function hangs on infinite lists because `dropWhileEnd` hangs on such lists ♾️